### ISSUE TITLE
Don't working on MIUI 14 Poco Global

### ISSUE THREAD

**Issue by Foxydroid** [Mods-Center/Miui_Launcher_Mod#12]
Doesn't work on Poco X3 Pro. The launcher disappears and the new one does not appear. That is, the system was without a launcher. MIUI 14.0.1.0 Global

**Comment by ahmetakif616** [comment]
@Foxydroid are you sure you correctly installed module?

**Comment by Foxydroid** [comment]
@ahmetakif616 Yes, I installed Core Patch, it doesn't work, and I tried patches from Lucky Patcher

**Comment by ahmetakif616** [comment]
@Foxydroid Selected true options on install right?

**Comment by DodoLeDev** [comment]
**This problem is caused by missing permissions on the module files.**
I've created a pull request to fix it here: #19

And if you don't want to wait indefinitely for it to be merged into a new release, [you can download the fixed version from my forked repo](https://github.com/DodoLeDev/Miui_Launcher_Mod/releases/tag/version-14.3-fixed)
